### ISSUE TITLE
MultiPower for metabolomics/lipidomics

### ISSUE THREAD

**Issue by Titan100** [ConesaLab/MultiPower#7]
Is it possible to make MultiPower usable for metabolomics/lipidomics?

**Comment by soniatarazona** [comment]
Yes, no problem! I guess that with a proper normalization we can consider that such data follows a gaussian distribution, so you can use type=2 for that.

Thanks!
